libcontainer/rootfs_linux.go

```diff
@@ -107,7 +116,10 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
 		if err := os.MkdirAll(dest, 0755); err != nil {
 			return err
 		}
-		return syscall.Mount(m.Source, dest, m.Device, uintptr(m.Flags), "")
+		if err := syscall.Mount(m.Source, dest, m.Device, uintptr(m.Flags), ""); err != nil {
+			return err
+		}
+		return setMountPropagation(dest, syscall.MS_PRIVATE|syscall.MS_REC)
```

Comment:
I put the comments in commit message that why are we making all these mounts PRIVATE.